server/src/modules-resolver.ts

```diff
@@ -12,3 +14,20 @@ export function findPathToModule(dir: string, moduleName: string): string|undefi
         return undefined;
     }
 }
+
+export function findPathToSdk(
+    dir: string,
+    moduleName: string
+): string | undefined {
```

Comment:
This is specific to yarn so let's please call it what it is since SDK is a very generic term and can be confusing in context of this repo.

Also no need for such aggressive line breaking. This would do :)
```suggestion
export function findPathToYarnSdk(dir: string, moduleName: string): string | undefined {
```

Reply:
👍 sorry, my default prettier setup. Would be good to get something like prettier included in the project to avoid issues like this

Reply:
Personal opinion but prettier is bad since it is very opinionated without letting the user customize how things should look much. So we're not gonna introduce it. The eslint rules, even if don't force all the style, should be enough.

I think that in general you should not be running prettier on foreign projects since there is a high chance that it will just not be in line what the project uses. If I'd be using it, I'd disable it by default and only enable on a per-project case (for my projects only).

Reply:
Yeah, I usually find myself in project with prettier so it's not normally a problem - I find it a really useful tool but that's probably because I'm a bit lazy 😄 

I can totally see why people don't like the opinions of prettier but I personally think it's better to have a tool that sets the opinion rather than trying to match up with anyone's opinion. By using a tool the conversation is only "have you run prettier" and ensuring the pipeline passes.